website/src/pages/components/form2/index.js

```diff
@@ -7,6 +7,21 @@ const scope = allDesignSystem;
 
 function FormPage() {
   const code = formatCode(`
+    const fruitOptions = [
```

Comment:
Can we have the form in `form2/index.js` mimic the one in `form/index.js`?
It would be nice to look at both side-by-side.